tests/helpers/searchHelper-test.js

```diff
@@ -19,8 +19,12 @@ test('areFiltersEqual should return false when filters are selected', () => {
     })).toBeFalsy();
 });
 
-test('isSearchHashReady knows when theres a hash', () => {
-    expect(isSearchHashReady('/search')).toEqual(false);
-    expect(isSearchHashReady('/search/')).toEqual(false);
-    expect(isSearchHashReady('/search/test')).toEqual(true);
+test.each([
+    ['', false],
+    ['/?', false],
+    ['/?fy=2020&period=12&hash=', false],
+    ['/?fy=2020&period=12&hash=t', true]
+])('when input (a query param) is %s return value is %s', (input, rtrn) => {
+    expect(isSearchHashReady({ search: input })).toEqual(rtrn);
 });
+
```

Comment:
`new URLSearchParams` is undefined whe node.process.ENV === test. I've got it mocked out elswhere so stuff doesn't break. But testing it would amount to rewriting that constructor.